### ISSUE TITLE
Docker only: acestream.conf not being used

### ISSUE THREAD

**Issue by sotiris-bos** [blaise-io/acelink#34]
Hello, the acestream.conf file is not being used by start-engine.

I suggest modifying the Dockerfile CMD to:

```
CMD /opt/acestream/start-engine --use-ffmpeg=1 --client-console @/opt/acestream/acestream.conf --bind-all
```

**Comment by blaise-io** [comment]
My approach so far was not using the conf file but instead just pass all non-default settings in the CMD args in the Dockerfile. 

Are there any settings in acestream.conf you're missing?

**Comment by sotiris-bos** [comment]
That way the user either needs to rebuild the container or override the CMD at the docker run level to modify the acestream configuration. I don't know if you provide an easier way to do that with Acelink, I am just using the docker only version on Linux to push acestreams to a TVHeadend server.

I am using compose and I mapped the acestream.conf file to an external file and tried to set a bandwidth limit only to find out that start-engine does not use the acestream.conf file. Hence I built the container myself with the change mentioned above.

Given that your true product here is acelink, this issue might be a bit off-topic and apply only to docker only users. I do think however providing an easy way to modify the acestream configuration/options is useful for the end user.
